app/presenters/migration/index.rb

```diff
@@ -5,16 +5,56 @@ class Index < ApplicationPresenter
     def content
       mustache[:content] ||= begin
         {
-          title: page_content_heading
+          channel_info: {
+            name: page_content_heading
+          },
+          title: page_content_heading,
+          hero: {
+            url:              '/images/channel_hero_migrations.jpg',
+            title:            t('hero.title'),
+            subtitle:         t('hero.subtitle'),
+            attribution_text: t('hero.attribution_text'),
+            attribution_url:  'http://www.europeana.eu/portal/record/2048211/449.html',
+            license_public:   true
+          },
+          description: t('description'),
+          closing_remark: t('closing_remark'),
+          begin_link: {
+            url: new_migration_path,
+            text: t('begin_link.text'),
+            text_long: t('begin_link.text_long')
+          },
+          call_to_action: call_to_action,
+          previews: [
```

Comment:
Instead of repeating this structure, create a method for assembling the Hash for one preview and call it multiple times.